source/camera/basic/CaptureTutorial.md

```diff
@@ -111,74 +108,79 @@ We may choose to configure settings manually. For more information about what ea
 
 We can configure settings for an individual frame directly to the camera ([go to source][settings-url]).
 ```python
-with camera.update_settings() as updater:
-    updater.settings.iris = 20
-    updater.settings.exposure_time = datetime.timedelta(microseconds=8333)
-    updater.setting.brightness = 1
-    updater.setting.gain = 1
-    updater.setting.bidirectional = False
-    updater.setting.filters.contrast.enabled = True
-    updater.setting.filters.contrast.threshold = 5
-    updater.setting.filters.gaussian.enabled = True
-    updater.setting.filters.gaussian.sigma = 1.5
-    updater.setting.filters.outlier.enabled = True
-    updater.setting.filters.outlier.threshold = 5
-    updater.setting.filters.reflection.enabled = True
-    updater.setting.filters.saturated.enabled = True
-    updater.setting.blue_balance = 1.081
-    updater.setting.red_balance = 1.709
+settings = zivid.Settings(
+    acquisitions=[
+        zivid.Settings.Acquisition(
+            aperture=5.66, exposure_time=datetime.timedelta(microseconds=8333),
+        ),
+    ],
+    processing=zivid.Settings.Processing(
+        filters=zivid.Settings.Processing.Filters(
+            outlier=zivid.Settings.Processing.Filters.Outlier(
+                removal=zivid.Settings.Processing.Filters.Outlier.Removal(
+                    enabled=True, threshold=5
+                )
+            )
+        )
+    ),
+)
 ```
 
-#### HDR Frame
+#### Multi Acquisition HDR
 
-We may also set a list of settings to be used in an [HDR capture](#capture-hdr) ([go to source][settingsHDR-url]).
+We may also create settings to be used in an HDR capture ([go to source][settings-hdr-url]).
 ```python
-settings_list = [camera.settings for _ in range(3)]
-settings_list[0].iris = 14
-settings_list[1].iris = 21
-settings_list[2].iris = 35
+settings = zivid.Settings(
+    acquisitions=[
+        zivid.Settings.Acquisition(
+            exposure_time=datetime.timedelta(microseconds=10000),
+        ),
+        zivid.Settings.Acquisition(
+            exposure_time=datetime.timedelta(microseconds=40000),
+        ),
+    ],
+)
```

Comment:
If you want something very clean you can consider something like this:
```
settings = zivid.Settings(
    acquisitions=[zivid.Settings.Acquisition(aperture=fnum) for fnum in (10.90, 5.80, 2.83)])
```

Reply:
This  method allows just one parameter to be used in a loop. If you want to extend it to change two parameters then you need to redesign the code. Therefore, I prefer the other method, where you can just add a comma and another parameter.

Reply:
Good point. I agree. Leave it as is :)